wandb/sdk_py27/internal/internal_api.py

```diff
@@ -738,6 +738,91 @@ def upsert_project(self, project, id=None, description=None, entity=None):
         #   'description': description, 'repo': self.git.remote_url, 'id': id})
         return response["upsertModel"]["model"]
 
+    @normalize_exceptions
+    def get_project_run_queues(self, entity, project):
```

Comment:
why are these fns in this pr? do you need to pull master

Reply:
No, we forgot to run codemod on the initial PR where we got rid of internal_runqueue.py